kubernetes/ingress/zooniverse.org.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: zooniverse-org-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+    nginx.ingress.kubernetes.io/set-real-ip-from: "10.0.0.0/8"
+spec:
+  tls:
+  - hosts:
+    - zooniverse.org
+    secretName: zooniverse-org-tls-secret
+  rules:
+  - host: zooniverse.org
+    http:
+      paths:
+      - backend:
+          serviceName: http-frontend
+          servicePort: 80
+        path: /(.*)
```

Comment:
OOI i assume if we don't specify the path regex here and use that in the `rewrite-target` we lose the path matching on the incoming requests?

Basically is this is needed if we're matching all paths and just passing to the upstream as is...?

Reply:
I'm pretty sure it's needed if I've understood the docs correctly.